post_blog.php

```diff
@@ -0,0 +1,34 @@
+<?php
+if(isset($_POST['post_blog'])){
+
+$title = $_POST['title'];
+$description = $_POST['description'];
+$content = $_POST['content'];
+
+	if (isset($_POST['checkboxvar'])) 
+{
+   $category = implode(' ', $_POST['checkboxvar']) ;
+   echo $category;
+}
+else {
+	$error = 'you need to select category';
+}
+echo ' '. $title . ' ' . $description . ' ' . $content;
+
+
+include 'connection.php';
+$data = new blogpost;
+
+$insert_array = array(
```

Comment:
ok na to. kelangan mo nlng tawaging ung blog_post->insert para makapag insert ka ng data